src/Resources/public/Admin.js

```diff
@@ -681,7 +681,12 @@ var Admin = {
         Admin.log('[core|setup_form_submit] setup form submit on', subject);
 
         jQuery(subject).find('form').on('submit', function() {
-            jQuery(this).find('button').prop('disabled', true);
+            var form = jQuery(this);
+
+            // this allows to submit forms and know which button was clicked
+            setTimeout(function() {
+                formform.find('button').prop('disabled', true);
```

Comment:
Using `setTimeout` feels like a hack, but if there is no other option, why not.

Reply:
That's why I left a comment there, to not remove it if don't find a better solution.